FISH-6391 Fix sonatype-2014-0173 commons-fileupload : commons-fileupload : 1.3.3

arieki opened the pull request:
Added security fixes list

JamesHillyard commented:
@arieki I have put the fix at the top of the table, fixed the Enterprise version (It was community before) and changed the reference to the sonatype issue - Can you confirm my changes look good to you?

arieki commented:
> @arieki I have put the fix at the top of the table, fixed the Enterprise version (It was community before) and changed the reference to the sonatype issue - Can you confirm my changes look good to you?

Yes it is, thanks a lot @JamesHillyard
